db/migrate/20150316174702_create_reviews.rb

```diff
@@ -0,0 +1,15 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.string :priority_id
+      t.string :status
```

Comment:
It should be a boolean, the difference is in review, it's either picked up or didn't.
in pickup the status is whether an email was sent to the user or not...